### DISCUSSION TITLE
Save files of games

### DISCUSSION THREAD

**Discussion by Dashhie550** [skyline-emu/skyline#855] [Q&A]
Where exactly can i find the save file of games? Everytime i install a new update all my game progress gets reset. Please implement the save file access feature so that i can back up my save files before installing a new update.

**Comment by D-06** [comment]
Now the save file is change to internal storage so you can backup it manually

**Comment by D-06** [comment]
To do this you need to go android/data/skyline.emu/files/switch/nand/user/save and backup that folder
For android 11 or up you can use zarchive or x-plore
Edit: reported stock explorer give some error when try to cut or copy. Recommend using the same as the android 11 or up

**Reply by SavXL** [reply]
I tried doing this and keep getting an error. Says the files aren't there when they actual are. Going in and trying to copy the files directly gives me the same error.
![Screenshot_20220411-050844_Solid Explorer](https://user-images.githubusercontent.com/87171080/162738001-c8e54d1f-86e8-4e1e-a340-e0146c293ae1.png)
Edit: Just had to not use Solid Explorer and it worked. Ended up using Zarchiver and it worked perfectly. Crisis averted.
